### ISSUE TITLE
Syntax error when hitting server

### ISSUE THREAD

**Issue by carsey** [directus/directus-docker#4]
After using `docker-compose up` to start the server, hitting `localhost:8080` is causing this PHP error:
```
Parse error: syntax error, unexpected '{' in /var/www/html/api/configuration.php on line 72
```

PHP version: 5.6
Directus version: 6.4

Line 72 of that file is
```
 'feedback' => [
        'token' => '{{feedback_token}}',
        'login' => {{feedback_login}}
    ],
```

**Comment by WellingGuzman** [comment]
Hey @carsey,

Thanks for reporting this, we have solved the problem and we are releasing 6.3.7 and use it in the docker, sorry for any inconveniences.

I will let you know as soon as it's pushed.

**Comment by WellingGuzman** [comment]
Hey @carsey I updated the docker image, please update the image and it should be fixed.

I'm closing this ticket, if the problem still happening I will re-open it.
